packages/server/src/routes/drivers/driver-routes.ts

```diff
@@ -0,0 +1,11 @@
+import * as express from "express";
+import { getDrivers, postDriver, deleteDriver, putDriver } from "../../controllers/driver/driver-controller";
+
+const router = express.Router();
+
+router.get("/:id", getDrivers);
+router.post("/", postDriver);
+router.delete("/:id", deleteDriver);
+router.put("/", putDriver);
```

Comment:
we dont have getDrivers to get them all ?

Reply:
Every user gets its related drivers(the drivers that the user adds). @hamzahmurrar